forc-plugins/forc-crypto/src/keygen/new_key.rs

```diff
@@ -0,0 +1,66 @@
+//! This file is migrated from https://github.com/FuelLabs/fuel-core/blob/master/bin/keygen/src/keygen.rs
+use super::KeyType;
+use anyhow::Result;
+use fuel_core_types::{
+    fuel_crypto::{
+        rand::{prelude::StdRng, SeedableRng},
+        SecretKey,
+    },
+    fuel_tx::Input,
+};
+use libp2p_core::{
+    identity::{secp256k1, Keypair},
+    PeerId,
+};
+use serde_json::json;
+use std::ops::Deref;
+
+/// Generate a random new secret & public key in the format expected by fuel-core
+#[derive(Debug, clap::Args)]
+#[clap(author, version, about = "Creates a new key for use with fuel-core")]
+pub struct Arg {
+    #[clap(long = "pretty", short = 'p')]
+    pretty: bool,
+    #[clap(
+        long = "key-type",
+        short = 'k',
+        value_enum,
+        default_value = "block-production"
```

Comment:
Is this supposed to be `block_production` instead of `block-production`?

Reply:
There seems to be inconsistencies here and the next file. Please double check which it's supposed to be.

Reply:
It seems to be `block_production` https://github.com/search?q=org%3AFuelLabs+block_production&type=code vs https://github.com/search?q=org%3AFuelLabs+block-production&type=code